client/src/components/History/HistoryViews/JobHistoryModal.jsx

```diff
@@ -146,14 +152,16 @@ function JobHistoryContent({ jobDetails, jobDisplay, jobName, toggle }) {
           }}
         />
         {isTerminalState && (
-          <Button
-            color="primary"
-            type="submit"
-            className={styles['submit-button']}
-            onClick={resubmitJob}
-          >
-            Resubmit Job
-          </Button>
+          <div className={styles['submit-button-container']}>
+            <Button
+              type="primary"
+              attr="submit"
+              size="large"
+              onClick={resubmitJob}
+            >
+              Resubmit Job
+            </Button>
+          </div>
```

Comment:
The container should not be necessary, but please let me know if I am not seeing the use case.
- Changed via 8e0a781 in #675.

Also, this button should stretch (new requirement, [FP-1733](https://jira.tacc.utexas.edu/browse/FP-1733)) to parent's width.
- Changed via 59d3897 in #675.